<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admGeneral</class>
 <widget class="QDialog" name="admGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">#label_barraNav{
background-color: rgb(0, 47, 76);
}

#label_tipoUsuario{
	font: 63 18pt &quot;Segoe UI Semibold&quot;;
	color: rgb(255, 255, 255,220);
}

#btnPerfil,#btnAlta,#btnEdicion, #btnCerrarSesion{
	background-color: none;
	color: rgb(255, 255, 255);
	font: 12pt &quot;Segoe UI Semilight&quot;;
	border-style:solid;
	text-align: left;
	padding-left:28px;
}

#btnPerfil:hover, #btnAlta:hover, #btnEdicion:hover, #btnCerrarSesion:hover
{
	background-color: rgb(255, 255, 255,80);
}</string>
  </property>
  <widget class="QLabel" name="label_barraNav">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>312</width>
     <height>771</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_tipoUsuario">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Admin. Área</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPerfil">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>312</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Perfil</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>1041</width>
     <height>771</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_perfil">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_alta">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Matricula</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tMatricula">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tNombre">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Edad</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sEdad">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>-140</y>
       <width>42</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>180</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>(Confirmación)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellido Materno</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tAmaterno">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Contraseña:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellido Paterno:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tApaterno">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tClave">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Celular:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Contraseña:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tCelular">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>140</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tCorreo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tClaveC">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>180</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConfirmar">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>90</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirmar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_edicion">
    <attribute name="title">
     <string>Tab 3</string>
    </attribute>
    <widget class="QTableWidget" name="lEmpleados">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>301</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Microsoft YaHei&quot;;</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Clave</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Area</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnActualizar">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>210</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Actualizar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cAM">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cNombre">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>110</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Matricula</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cAP">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>170</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Celular</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cCelular">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cCorreo">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>170</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>140</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellido Paterno</string>
     </property>
    </widget>
    <widget class="QLabel" name="cMatricula">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cMatricula</string>
     </property>
    </widget>
    <widget class="QLabel" name="cEdad">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>cEdad</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>170</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Edad</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>140</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellido Materno</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnAlta">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>312</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Alta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEdicion">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Edición</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCerrarSesion">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>700</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar Sesión</string>
   </property>
  </widget>
  <zorder>label_barraNav</zorder>
  <zorder>label_tipoUsuario</zorder>
  <zorder>btnPerfil</zorder>
  <zorder>line</zorder>
  <zorder>btnAlta</zorder>
  <zorder>btnEdicion</zorder>
  <zorder>tabWidget</zorder>
  <zorder>btnCerrarSesion</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
